warehouse/templates/manage/project/settings.html

```diff
@@ -102,7 +102,7 @@ <h3>{% trans %}<abbr title="two factor authentication">2FA</abbr> requirement{%
     <h3>{% trans %}Project description and sidebar{% endtrans %}</h3>
     <p>
       {% trans trimmed project_name=project.name, setup_args_href='https://packaging.python.org/tutorials/distributing-packages/#setup-args', twine_docs_href='https://twine.readthedocs.io/en/latest/', distribution_href='https://packaging.python.org/tutorials/distributing-packages/' %}
```

Comment:
We should probably also update the `setup_args_href` link to have a different name and point to https://packaging.python.org/en/latest/tutorials/packaging-projects/#creating-pyproject-toml or something similar instead.